### ISSUE TITLE
refgenie pull all assets for a genome

### ISSUE THREAD

**Issue by michalstolarczyk** [databio/refgenie#27]
raised in a discussion with @nsheff 

in lieu of combined tarballs of all assets for a genome (https://github.com/databio/refgenieserver/issues/18#issuecomment-497099152) we need a way to pull _all_ the assets for a genome. 

Is it `refgenie pull -g <genome> -a all`?

**Comment by nsheff** [comment]
maybe we don't *want* to enable this...

**Comment by michalstolarczyk** [comment]
are you worried about misusage of this functionality? I think it's pretty useful

**Comment by nsheff** [comment]
yeah, a bit worried...

**Comment by nsheff** [comment]
Hey @stolarczyk -- in conversation with @natefoo, this would be a useful feature for integration with galaxy, as they'd like to mirror a refgenieserver... (just download everything).

So, even something like `refgenie --mirror`, which would only download things from the server that aren't available locally. what do you think?

**Comment by stolarczyk** [comment]
so this would go even beyond a single genome? 

Yes, I'm still on board, this use case warrants implementation

**Comment by nsheff** [comment]
> so this would go even beyond a single genome?

Yeah. that would be everything. how hard would it be to implement `-all` and `--mirror`?

**Comment by stolarczyk** [comment]
I think this will be relatively easy:
1. implement "force _no_ overwrite" functionality in `RefGenConf.pull`
2. API call to this endpoint: http://refgenomes.databio.org/v2/assets
3. call `RefGenConf.pull(..., no_overwrite=True)` for each result
